src/nav/AppNavigator.js

```diff
@@ -45,6 +46,7 @@ export default createStackNavigator(
     'emoji-picker': { screen: EmojiPickerScreen },
     loading: { screen: LoadingScreen },
     main: { screen: MainScreenWithTabs },
+    'message-reaction': { screen: MessageReactionList },
```

Comment:
How about `message-reactions`? This is after all for *all* the reactions on a message, which is very often several of them.

(OTOH `MessageReactionList` is just fine -- it is indeed a "reaction list", i.e. a list of reactions.)